### PR TITLE
Update some scheduler misc

### PR DESCRIPTION
**Pull request by astraw99**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
To make the code more clean, and update some useful doc.

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: pkg/scheduler/profile/profile.go [hunk 1]
```diff
@@ -98,6 +98,7 @@ func (v *cfgValidator) validate(cfg config.KubeSchedulerProfile, f framework.Fra
 	for _, plCfg := range cfg.PluginConfig {
 		if plCfg.Name == queueSort {
 			queueSortArgs = plCfg.Args
+			break
```
Comment (ahg-g) [inline pkg/scheduler/profile/profile.go:101]: why this?

Comment (kerthcet) [reply pkg/scheduler/profile/profile.go:101]: Find the matched queueSortArgs and quick return. We can have only one pluginConfig for each plugin.

Comment (astraw99) [reply pkg/scheduler/profile/profile.go:101]: Yes, here is to validate the `queueSort` pluginConfig, add `break` to quick return.

**Pull request merged**